code/modules/client/preference_setup/loadout/lists/earwear.dm

```diff
@@ -4,7 +4,3 @@
 	display_name = "earmuffs"
 	path = /obj/item/clothing/ears/earmuffs
 	sort_category = "Earwear"
-
-/datum/gear/ears/headphones
```

Comment:
They aren't play music....

Reply:
Old headphone code was removed and replaced with mp3 headphones in a previous PR, which is gutted out in this PR.